### PR TITLE
[tests/e2e/ledger] add a test for many txs applied in the same block

### PR DESCRIPTION
**Pull request by tzemanovic**

### CODE REVIEW

File: tests/src/e2e/ledger_tests.rs [hunk 1]
```diff
@@ -401,3 +402,74 @@ fn invalid_transactions() -> Result<()> {
     client.assert_success();
     Ok(())
 }
+
+/// Test that multiple txs submitted in the same block all get the tx result.
+///
+/// In this test we:
+/// 1. Run the ledger node with 10ms consensus timeout
```
Comment (batconjurer) [inline tests/src/e2e/ledger_tests.rs:409]:
```suggestion
/// 1. Run the ledger node with 10s consensus timeout
```

Comment (tzemanovic) [reply tests/src/e2e/ledger_tests.rs:409]: I did it again T.T

**Review by batconjurer — APPROVED** [review]
Other than a tiny nit, LGTM

File: tests/src/e2e/ledger_tests.rs (hunk 1 above)
Comment (batconjurer) [reply tests/src/e2e/ledger_tests.rs]: I guess you just love living life in the fast lane :rofl:

**Pull request merged**